tests/e2e/cucumber/common-steps.spec.ts

```diff
@@ -52,9 +52,12 @@ When('navigating back', () => {
 });
 
 // Requests
-Then('search request contains the origin {string} in the URL', (origin: string) => {
-  cy.wait('@interceptedResults').its('request.url').should('contain', `origin=${origin}`);
-});
+Then(
```

Comment:
What do you think about changing this to something like:

```suggestion
Then('a search request from {string} is done', (origin: string) => {
  cy.wait(`@interceptedResultsFrom:${origin}`);
});
```

That way we avoid repeating the origin, and the interceptor is filtering it by origin anyway.